src/main/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorUtil.groovy

```diff
@@ -74,16 +70,17 @@ class IqPolicyEvaluatorUtil
 
       return evaluationResult
     }
-    catch (IqNetworkException e) {
+    catch (IqClientException e) {
       return handleNetworkException(iqPolicyEvaluator.failBuildOnNetworkError, e, listener, run)
```

Comment:
thoughts on multiple return statements in a method?  I usually try to avoid as they seem to make the code more fragile and more difficult to maintain

Reply:
Put it up to @bigspotteddog @fernau for discussion, IMHO managing a variable to capture the result of the catch block is more difficult than returning within it.

Reply:
I agree that managing a variable to capture the result and return at the bottom of the function is more difficult. I prefer to allow multiple return statements for simplicity.

Multiple return statements are a problem if the functions get tall enough to not be entirely visible on the screen. Our functions will never be long enough for that to be a problem.